### ISSUE TITLE
Feature Request: Build ML Diagnosis Engine / Model for Health Prediction

### ISSUE THREAD

**Issue by riyajoshi2004** [aayushraj1010/med-genie#179]
I was going through the roadmap of Med-Genie and saw the “ML Diagnosis Engine / Advanced AI-driven health issue prediction” listed under future features.

Currently, I noticed that no issue has been raised and no one is working on building this model yet. I’d like to work on implementing it.

What I’d need / plan:

A public health-dataset (if project maintainers are okay with using open data)

Define metrics: accuracy, precision, recall etc.

Implement model (could be in Python / TensorFlow / PyTorch / scikit-learn)

Create API endpoint to serve the model predictions

If you think this is useful, I’d love to take this feature on. Please assign this request to me, or tell me how you’d prefer me to begin (a draft plan, or start building a prototype).

Thanks

**Comment by riyajoshi2004** [comment]
I have raised this issue because i have an experience on machine learning . Please assign this to me. I will provide the result within a week.

**Comment by riyajoshi2004** [comment]
please assign this issue to me.

**Comment by Kavish-Paraswar** [comment]
@aayushraj1010 why was i unassigned , can ik the reason?

**Comment by Kavish-Paraswar** [comment]
@riyajoshi2004 how much work have you done , we can collaboarate on it!

**Comment by Krishnadev-sharma** [comment]
@aayushraj1010  hi want to work on this issue. Please assign this issue to me.
